src/ordered_hash_set.py

```diff
@@ -21,6 +21,12 @@ def add(self, item):
             if len(self._items) != 1:
                 self._items[self._last][1][1] = item
             self._last = item
+            return self.__len__() - 1
+        else:
+            temp = self.get_all()
+            for i in range(self.__len__()):
+                if temp[i] == item:
+                    return i
```

Comment:
Since get_all returns a list, you can use [index](https://www.programiz.com/python-programming/methods/list/index) method instead of iterating over the list items